.golangci.yml

```
@@ -106,3 +106,15 @@ linters-settings:
       - golang.org/x/net/context:
           recommendations:
           - context
+      - github.com/kong/kubernetes-ingress-controller/v3:
+          recommendations:
+          - github.com/kong/kubernetes-configuration
+      - github.com/kong/gateway-operator:
+          recommendations:
+          - github.com/kong/kubernetes-configuration
+  importas:
+    no-unaliased: true
+    alias:
+    - pkg: github.com/kong/kubernetes-configuration/api/configuration/(v[\w\d]+)
+      alias: kc${1}
```

For consistency's sake, I suggest using the same alias as we do in KGO.
```suggestion
      alias: configuration${1}
```

---

Ok, I'm changing it, but TBH, this alias `configuration` makes names unnecessary long and unconvenient to use in function signatures, etc. Maybe something to discuss during team-sync...